### PR TITLE
Feature/icaps vrdip

### PR DESCRIPTION
**Pull request by pmandler**
There's an option for the author to be anonymous.  I have it at the bottom of the form now.  Think I should move it up to just below the email addresses?

### CODE REVIEW

File: venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init_conference.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
```
Comment (melisabok) [inline venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init_conference.py]: For convention we name the init file as 'init.py'

Comment (pmandler) [reply venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init_conference.py]: I've changed the file name.  By moving the submission content anonymity up to 4, do I have to change the order number on all of the other content fields in order to get it to show up near the top?

Comment (pmandler) [reply venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init_conference.py]: Ready to merge?

File: venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        "anonymized": {
+            "order": 4,
+            "value-checkbox": "Anonymize author identities",
+            "required": False
+        }
+    })
+
+# doesn't seem to take the order information during init
+invite = client.get_invitation(id=conference.get_submission_id())
+invite.reply['content']['anonymized']['order'] = 4
```
Comment (melisabok) [inline venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init.py]: You are right, I will create an issue to fix this.

**Pull request merged**